test/ignore-errors.ts

```diff
@@ -24,8 +24,16 @@ const reprojectAndZarrQuery = {
   concatenate: false,
 };
 
+const l2ssAndConciseQuery = {
+  subset: 'lat(0:90)',
+  concatenate: false,
+  maxResults: 1,
+  ignoreErrors: true,
+};
+
 describe('when setting ignoreErrors=true', function () {
   const collection = 'C1233800302-EEDTEST';
+  const l2ssCollection = 'C1243729749-EEDTEST';
```

Comment:
Thanks for finding another EEDTEST collection for us to use and sorry it required so many updates... there's a lot of branching in the code but clearly a little too much repetition in the tests!